### ISSUE TITLE
require(electron) doesn't work with eel.start

### ISSUE THREAD

**Issue by CyberSinister** [samuelhwilliams/Eel#148]
My code: 
``` js
const electron = require('electron');
const url = require('url');
const path = require('path');

const {app, BrowserWindow, Menu} = electron;

let mainWindow;

//Listen for app to be ready
app.on('ready', function(){
    //Create new window
    mainWindow = new BrowserWindow({});
    //Load HTML into window
    mainWindow.loadURL(url.format({
        pathname: path.join(__dirname, 'index.html'),
        protocol:'file',
        slashes: true
    }));
    //Using the menu Template we made down outside this funciton
    const mainMenu = Menu.buildFromTemplate(menuTemplate);
    //Insert the Menu
    Menu.setApplicationMenu(mainMenu);
});

//Creating a menu template.
//This is what shows up on the menubar on top.
const menuTemplate = [
    {
        label: 'Connection Management',
        submenu:[
            {
                label: 'Start Listening',
                click(){
                    dumbpy();
                }
            },
            {
                label: 'Stop Listening'
            },
            {
                label: 'EDAAC'
            }
        ]
    }
]
```

The menu only shows when I do `npm start` in the director. It doesn't when I run the `.py` file. When I open the the inspect element, I get this : `Uncaught ReferenceError: require is not defined at main.js:1`

**Comment by 04fsnape** [comment]
Seems to be the same issue I am having [here](https://github.com/ChrisKnott/Eel/issues/147).

**Comment by CyberSinister** [comment]
Right. This is really annoying but I'm sure Chris or someone else has a solution for this.

**Comment by ChrisKnott** [comment]
Hi guys, I will post an example Electron demo this evening that you can copy

**Comment by 04fsnape** [comment]
@ChrisKnott Sorry to be impatient, but are you busy or are you having issues getting it to work? I just wasn't sure if I should sit here or try get it working myself.

**Comment by CyberSinister** [comment]
Okay. Looks like Chris is a bit too busy.

**Comment by ChrisKnott** [comment]
Hi guys, sorry yes I was very busy at work. I am on holiday now so have some time for coding 😁

I will get an example up ASAP 👍

**Comment by CyberSinister** [comment]
All good brother. Thanks in advance.

**Comment by ChrisKnott** [comment]
@CyberSinister can you check out latest and try the new example, see if you get the same issue with that

**Comment by CyberSinister** [comment]
OMG. I am so sorry for such a late reply. I had gotten sick but I'm better now. Yes I've checked the example you uploaded and I'm guessing that it's because I didn't put in the options line in my Python code. However, Now that I did put it in, it returns an error saying that there is no 'electron.exe' file in that location. I have checked the exact path manually myself and saw no "electron.exe" file in that location. I of the node_modules folders. I am using `LINUX` and the .py file is in the same directory as the `node_modules` folder. What am I missing here? Is it cause I'm on Linux?
